<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/tool/image/server.png</normaloff>:/tool/image/server.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>800</width>
         <height>535</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="homePage">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QScrollArea" name="scrollArea_4">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_3">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>800</width>
                 <height>535</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>10</number>
                </property>
                <property name="topMargin">
                 <number>10</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QListWidget" name="listTransmission">
                  <property name="font">
                   <font>
                    <family>Times New Roman</family>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="autoScroll">
                   <bool>true</bool>
                  </property>
                  <property name="dragEnabled">
                   <bool>false</bool>
                  </property>
                  <property name="dragDropOverwriteMode">
                   <bool>false</bool>
                  </property>
                  <property name="selectionMode">
                   <enum>QAbstractItemView::NoSelection</enum>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="sortingEnabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="videoPage">
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="8,1,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QScrollArea" name="Video">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustIgnored</enum>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_2">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>800</width>
                 <height>365</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4"/>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QScrollArea" name="scrollArea_3">
              <property name="styleSheet">
               <string notr="true">border:none;</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_6">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>800</width>
                 <height>85</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="timePosition">
                  <property name="font">
                   <font>
                    <family>Times New Roman</family>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>0:00</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="timeSlider">
                  <property name="pageStep">
                   <number>3</number>
                  </property>
                  <property name="sliderPosition">
                   <number>0</number>
                  </property>
                  <property name="tracking">
                   <bool>false</bool>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="tickPosition">
                   <enum>QSlider::NoTicks</enum>
                  </property>
                  <property name="tickInterval">
                   <number>10</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="timeMaxVideo">
                  <property name="font">
                   <font>
                    <family>Times New Roman</family>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>00:00:00</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QScrollArea" name="scrollArea_2">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_5">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>800</width>
                 <height>85</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,5,3">
                <property name="spacing">
                 <number>5</number>
                </property>
                <property name="leftMargin">
                 <number>20</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>20</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QWidget" name="widget_4" native="true">
                  <property name="styleSheet">
                   <string notr="true">border:none;</string>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="8,0,0">
                   <property name="spacing">
                    <number>5</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QToolButton" name="buttonOpen">
                     <property name="styleSheet">
                      <string notr="true"/>
                     </property>
                     <property name="text">
                      <string>...</string>
                     </property>
                     <property name="icon">
                      <iconset resource="image.qrc">
                       <normaloff>:/tool/image/open-folder.png</normaloff>:/tool/image/open-folder.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>45</width>
                       <height>45</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QToolButton" name="buttonVolume">
                     <property name="text">
                      <string>...</string>
                     </property>
                     <property name="icon">
                      <iconset resource="image.qrc">
                       <normaloff>:/tool/image/volume-up-down.png</normaloff>
                       <normalon>:/tool/image/volume-up.png</normalon>:/tool/image/volume-up-down.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>45</width>
                       <height>45</height>
                      </size>
                     </property>
                     <property name="checkable">
                      <bool>true</bool>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSlider" name="horizontalSlider_2">
                     <property name="maximum">
                      <number>100</number>
                     </property>
                     <property name="value">
                      <number>50</number>
                     </property>
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="tickPosition">
                      <enum>QSlider::TicksAbove</enum>
                     </property>
                     <property name="tickInterval">
                      <number>10</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_3" native="true">
                  <property name="styleSheet">
                   <string notr="true">border:none;
</string>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <property name="spacing">
                    <number>5</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QToolButton" name="beforeVideo">
                     <property name="text">
                      <string>...</string>
                     </property>
                     <property name="icon">
                      <iconset resource="image.qrc">
                       <normaloff>:/tool/image/back.png</normaloff>:/tool/image/back.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>45</width>
                       <height>45</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QToolButton" name="videoTranslationLeft">
                     <property name="text">
                      <string>...</string>
                     </property>
                     <property name="icon">
                      <iconset resource="image.qrc">
                       <normaloff>:/tool/image/next-button-left.png</normaloff>:/tool/image/next-button-left.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>45</width>
                       <height>45</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QToolButton" name="toolButton">
                     <property name="text">
                      <string>...</string>
                     </property>
                     <property name="icon">
                      <iconset resource="image.qrc">
                       <normaloff>:/tool/image/play.png</normaloff>
                       <normalon>:/tool/image/pause.png</normalon>:/tool/image/play.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>45</width>
                       <height>45</height>
                      </size>
                     </property>
                     <property name="checkable">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QToolButton" name="videoTranslationRight">
                     <property name="text">
                      <string>...</string>
                     </property>
                     <property name="icon">
                      <iconset resource="image.qrc">
                       <normaloff>:/tool/image/next-button.png</normaloff>:/tool/image/next-button.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>45</width>
                       <height>45</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QToolButton" name="NextVideo">
                     <property name="text">
                      <string>...</string>
                     </property>
                     <property name="icon">
                      <iconset resource="image.qrc">
                       <normaloff>:/tool/image/skip.png</normaloff>:/tool/image/skip.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>45</width>
                       <height>45</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget" native="true">
                  <property name="styleSheet">
                   <string notr="true">border:none;</string>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <property name="spacing">
                    <number>5</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QToolButton" name="toolButton_3">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="image.qrc">
                       <normaloff>:/tool/image/speed.png</normaloff>:/tool/image/speed.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>45</width>
                       <height>45</height>
                      </size>
                     </property>
                     <property name="checkable">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSlider" name="videoSpeed">
                     <property name="minimum">
                      <number>25</number>
                     </property>
                     <property name="maximum">
                      <number>200</number>
                     </property>
                     <property name="singleStep">
                      <number>25</number>
                     </property>
                     <property name="pageStep">
                      <number>50</number>
                     </property>
                     <property name="value">
                      <number>100</number>
                     </property>
                     <property name="sliderPosition">
                      <number>100</number>
                     </property>
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="tickPosition">
                      <enum>QSlider::NoTicks</enum>
                     </property>
                     <property name="tickInterval">
                      <number>10</number>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QToolButton" name="buttonLoop">
                     <property name="text">
                      <string>...</string>
                     </property>
                     <property name="icon">
                      <iconset resource="image.qrc">
                       <normaloff>:/tool/image/reload-off.png</normaloff>
                       <normalon>:/tool/image/reload.png</normalon>:/tool/image/reload-off.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>45</width>
                       <height>45</height>
                      </size>
                     </property>
                     <property name="checkable">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="livePage"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Thanh công cụ</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionServer"/>
   <addaction name="actionVideo"/>
   <addaction name="actionLive"/>
   <addaction name="actionDataTransmission"/>
  </widget>
  <action name="actionServer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/tool/image/power-off.png</normaloff>
     <normalon>:/tool/image/power-on.png</normalon>:/tool/image/power-off.png</iconset>
   </property>
   <property name="text">
    <string>Server</string>
   </property>
  </action>
  <action name="actionLive">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/tool/image/live-streaming-off.png</normaloff>
     <normalon>:/tool/image/live-streaming-on.png</normalon>:/tool/image/live-streaming-off.png</iconset>
   </property>
   <property name="text">
    <string>Live</string>
   </property>
  </action>
  <action name="actionVideo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/tool/image/video-off.png</normaloff>
     <normalon>:/tool/image/video.png</normalon>:/tool/image/video-off.png</iconset>
   </property>
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="actionDataTransmission">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/tool/image/upload.png</normaloff>:/tool/image/upload.png</iconset>
   </property>
   <property name="text">
    <string>dataTransmission</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
